playground/backend/cmd/server/http.go

```diff
@@ -33,3 +49,28 @@ func listenHttp(ctx context.Context, errChan chan error, envs environment.Networ
 		return
 	}
 }
+
+// isReady checks the number of already working code processing.
+//  It counts by the number of the /path/to/workingDir/executable_files/{pipelineId} folders.
+// If it is equals or more than numOfParallelJobs, then returns false.
+// If it is less than numOfParallelJobs, then returns true.
+func isReady(workingDir string, numOfParallelJobs int) bool {
```

Comment:
I believe that server/http not a best place for business logic

Reply:
Done.